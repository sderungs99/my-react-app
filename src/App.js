import React, { Component } from 'react';
import './App.css';
import Header from './components/common/Header';
import HomePage from './components/home/HomePage';

class App extends Component {
    render() {
        return (
            <div className="App">
                <Header />
                <HomePage />
            </div>
        );
    }
}

export default App;
