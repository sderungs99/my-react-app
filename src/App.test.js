import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import App from './App';

function setup(props) {
    return shallow(<App {...props} />);
}

it('renders without crashing', () => {
    setup();
});

it('renders as expected', () => {
    const component = renderer.create(
        <App />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});