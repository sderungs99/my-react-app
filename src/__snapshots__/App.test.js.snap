// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders as expected 1`] = `
<div
  className="App"
>
  <div
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="logo.svg"
    />
    <h2>
      Welcome to React
    </h2>
  </div>
  <div>
    <p
      className="App-intro"
    >
      To get started, edit 
      <code>
        src/App.js
      </code>
       and save to reload.
    </p>
    <p>
      <button
        onClick={[Function]}
      >
        Test Button
      </button>
    </p>
    <p>
      Click Counter: 
      0
    </p>
  </div>
</div>
`;
